### PR TITLE
change watson conversation name to watson assistant

### PR DESCRIPTION
**Pull request by stevemart**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 'use strict';
 
-var WatsonConversationSetup = require('./lib/watson-conversation-setup');
+var WatsonAssistant for BusinessSetup = require('./lib/watson-conversation-setup');
```
Comment (scottdangelo) [inline app.js]: This is breaking Travis CI.
Can't have a 3 word variable.

File: app.js [hunk 2]
```diff
@@ -45,14 +45,14 @@ var conversation = watson.conversation({
   version: 'v1'
 });
 
-var conversationSetup = new WatsonConversationSetup(conversation);
+var conversationSetup = new WatsonAssistant for BusinessSetup(conversation);
```
Comment (scottdangelo) [inline app.js]: same

File: app.js [hunk 3]
```diff
@@ -45,14 +45,14 @@ var conversation = watson.conversation({
...
 var workspaceJson = JSON.parse(fs.readFileSync('data/watson-pizzeria.json'));
 var conversationSetupParams = { default_name: DEFAULT_NAME, workspace_json: workspaceJson };
-conversationSetup.setupConversationWorkspace(conversationSetupParams, (err, data) => {
+conversationSetup.setupAssistant for BusinessWorkspace(conversationSetupParams, (err, data) => {
```
Comment (scottdangelo) [inline app.js]: same

File: README.md [hunk 4]
```diff
@@ -279,8 +279,8 @@ To disable tracking, simply remove ``cf_deployment_tracker.track()`` from the
 # Links
 
 * [Demo on youtube](https://youtu.be/6QlAnqSiWvo)
-* [IBM Cloud Conversation Docs](https://console.bluemix.net/docs/services/conversation/dialog-build.html#dialog-build)
-* [Blog for Conversation Slots Code Pattern](https://developer.ibm.com/code/2017/09/19/managing-resources-efficiently-watson-conversation-slots/)
+* [IBM Cloud Assistant Docs](https://console.bluemix.net/docs/services/conversation/dialog-build.html#dialog-build)
```
Comment (stevemart) [inline README.md]: Can we change this to "IBM Watson Assistant Docs"

File: app.js [hunk 5]
```diff
@@ -16,7 +16,7 @@
 
 'use strict';
 
-var WatsonConversationSetup = require('./lib/watson-conversation-setup');
+var WatsonAssistantSetup = require('./lib/watson-conversation-setup');
```
Comment (scottdangelo) [inline app.js]: We should not change this now.

File: app.js [hunk 6]
```diff
@@ -45,14 +45,14 @@ var conversation = watson.conversation({
   version: 'v1'
 });
 
-var conversationSetup = new WatsonConversationSetup(conversation);
+var conversationSetup = new WatsonAssistantSetup(conversation);
```
Comment (scottdangelo) [inline app.js]: same, don't change.

**Pull request merged**